xbmc/settings/dialogs/GUIDialogSettingsBase.cpp

```diff
@@ -521,13 +521,20 @@ std::set<std::string> CGUIDialogSettingsBase::CreateSettings()
     const CSettingControlTitle *title = dynamic_cast<CSettingControlTitle *>((*groupIt)->GetControl());
     bool hideSeparator = title ? title->IsSeparatorHidden() : false;
```

Comment:
Thought to allow separator on first if defined as above, to allow bars or something else.